src/main/java/de/slimecloud/slimeball/features/level/card/badge/BadgeCommand.java

```diff
@@ -40,7 +40,7 @@ public static void handleAutocomplete(@NotNull SlimeBot bot, @NotNull CommandAut
 
 	@Setup
 	public static void setup(@NotNull SlimeBot bot, @NotNull ListManager<ICommandContext> manager, @NotNull AnnotatedCommand<?, ICommandContext, ?> cmd) {
-		cmd.addSubcommand(manager.createCommand(state -> bot.getCardBadges()).withDescription("Zeigt alle Badges an"));
+		cmd.addSubcommand(manager.createCommand(state -> bot.getCardBadges()).withDescription("Zeigt alle Badges an").withCondition(CommandPermission.EVERYONE));
```

Comment:
We said `/badge list` should not be accessible to all but `/badge list_badges`, didn't we?